extension/src/popup/ducks/access.ts

```diff
@@ -4,7 +4,7 @@ import {
   rejectAccess as internalRejectAccess,
   grantAccess as internalGrantAccess,
   signTransaction as internalSignTransaction,
-} from "@lyra/api/internal";
+} from "@shared/api/internal";
 
```

Comment:
whoops - caught this while making this change

Reply:
Possibly not based off latest master? Should have been fixed in https://github.com/stellar/lyra/pull/60/commits/dba296eb6b4db547a4a0e0b948c5b8bde1d2f74f

Reply:
Oh that was only merged like 2 hours ago, so yeah make sense

Reply:
oh maybe i gotta merge